apps/dashboard/src/components/welcome/progress-section.tsx

```diff
@@ -0,0 +1,149 @@
+        className={`${step.status === 'completed' ? 'bg-success' : 'shadow-xs'} flex h-6 w-6 min-w-6 items-center justify-center rounded-full`}
+      >
+        {step.status === 'completed' ? (
+          <RiCheckLine className="h-4 w-4 text-[#ffffff]" />
+        ) : (
+          <RiLoader3Line className="text-foreground-400 h-4 w-4" />
+        )}
+      </div>
+
+      <Link
+        to={getStepRoute(step.id, environmentSlug).path}
+        reloadDocument={getStepRoute(step.id).isLegacy}
+        className="w-full"
+        onClick={handleStepClick}
+      >
+        <Card
+          className={`shadow-xs w-full p-1 ${step.status !== 'completed' ? 'transition-all duration-200 hover:translate-x-[1px] hover:shadow-md' : ''}`}
+        >
+          <CardContent className="flex flex-col rounded-[6px] bg-[#FBFBFB] px-2 py-1.5">
+            <div className="flex items-center justify-between">
+              <span
+                className={`text-xs ${step.status === 'completed' ? 'text-foreground-400 line-through' : 'text-foreground-600'}`}
+              >
+                {step.title}
+              </span>
+              <RiArrowRightDoubleFill className="text-foreground-400 h-4 w-4" />
+            </div>
+            <p className="text-foreground-400 text-[10px] leading-[14px]">{step.description}</p>
+          </CardContent>
+        </Card>
+      </Link>
+    </motion.div>
+  );
+}
+
+function WelcomeHeader() {
+  return (
+    <motion.div
+      variants={leftSection}
+      className="flex w-full max-w-[380px] grow flex-col items-start justify-between gap-2 rounded-l-xl bg-[#FBFBFB] p-6"
+    >
+      <div className="flex w-full flex-col gap-2">
+        <motion.h2 variants={textItem} className="font-label-medium text-base font-medium">
+          You're doing great work! 💪
+        </motion.h2>
+
+        <div className="text-foreground-400 flex flex-col gap-6 text-sm">
+          <motion.p variants={textItem}>Set up Novu to send notifications your users will love.</motion.p>
+          <motion.p variants={textItem}>
+            Streamline all your customer messaging in one tool and delight them at every touchpoint.
+          </motion.p>
+        </div>
+      </div>
+
+      <motion.div variants={textItem} className="space-between flex items-center gap-0">
+        <p className="text-foreground-400 text-sm">Get started with our setup guide.</p>
+        <PointingArrow className="relative left-[15px] top-[-10px]" />
+      </motion.div>
+    </motion.div>
+  );
+}
+
+function getStepRoute(stepId: StepIdEnum, environmentSlug: string = '') {
+  switch (stepId) {
+    case StepIdEnum.CREATE_A_WORKFLOW:
+      return {
+        path: buildRoute(ROUTES.WORKFLOWS, { environmentSlug }),
+        isLegacy: false,
+      };
+    case StepIdEnum.CONNECT_EMAIL_PROVIDER:
+    case StepIdEnum.CONNECT_IN_APP_PROVIDER:
+    case StepIdEnum.CONNECT_PUSH_PROVIDER:
+    case StepIdEnum.CONNECT_CHAT_PROVIDER:
+    case StepIdEnum.CONNECT_SMS_PROVIDER:
```

Comment:
What happens when users chooses Inbox + something? We should show show 2 steps here